submission/persistence-mongo/src/test/kotlin/ac/uk/ebi/biostd/persistence/doc/service/SubmissionMongoQueryServiceTest.kt

```diff
@@ -186,6 +188,52 @@ internal class SubmissionMongoQueryServiceTest(
             assertThat(result.second().accNo).isEqualTo("accNo3")
         }
 
+        @Test
+        fun `when all`() {
+            saveAsRequest(fullExtSubmission.copy(
+                accNo = "accNo1",
+                version = 1,
```

Comment:
In order to make the test a bit closer to reality, this should be the second version

Reply:
not possible because test intention is to validate all filters work together. So if change this version we can not test that